shared/date/date.properties.po

```diff
@@ -1,246 +1,244 @@
-#. # Day names
-#. extracted from ./shared/date/date.properties
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2015-02-11 19:45+0000\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2015-06-18 20:24+0200\n"
 "Language: nd\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Translate Toolkit 1.13.0-alpha1\n"
+"X-Generator: Poedit 1.8.1\n"
 "X-Accelerator-Marker: &\n"
 "X-Merge-On: location\n"
+"Last-Translator: \n"
+"Language-Team: \n"
 
 #: weekday-0-long
 msgid "Sunday"
-msgstr ""
+msgstr "Isonto"
 
 #: weekday-1-long
 msgid "Monday"
-msgstr ""
+msgstr "Umvulo"
 
 #: weekday-2-long
 msgid "Tuesday"
-msgstr ""
+msgstr "Olwesibili"
```

Comment:
Hmm this is an interesting case, is it "Ulwesibili" or "Olwesibili" My Ndebele teacher used to say its "Ulwesibili" as in *U*suku lwesibili. his qustion was nxa usithi "olwe" uyabe usithini vele?